app/backfill-synonym-slugs.js

```diff
@@ -0,0 +1,72 @@
+import { DynamoDBClient, paginateScan } from '@aws-sdk/client-dynamodb'
+import { GetParameterCommand, SSMClient } from '@aws-sdk/client-ssm'
+import { BatchWriteCommand } from '@aws-sdk/lib-dynamodb'
+import { unmarshall } from '@aws-sdk/util-dynamodb'
+import { slug } from 'github-slugger'
+
+function* chunks(arr, n) {
+  for (let i = 0; i < arr.length; i += n) {
+    yield arr.slice(i, i + n)
+  }
+}
```

Comment:
This is probably correct, but I'd use [lodash's `chunk`](https://lodash.com/docs/#chunk).